setup.cfg

```diff
@@ -32,4 +32,4 @@ packages = find:
 zip_safe = False
 python_requires = >=3.7
 install_requires =
-    aiohttp == 3.8.3
```

Comment:
for what? any bugs with the fixed version?

Reply:
to allow the usage of latest version of aiohttp